requirements.txt

```diff
@@ -1,2 +1,12 @@
+certifi==2020.6.20
+chardet==3.0.4
+click==7.1.2
+Flask==1.1.2
```

Comment:
You have two entries for Flask now, which breaks the file, and did you deliberately remove python-dotenv?

It's used for e.g. automatic loading of the .env file when running `flask run`.

Reply:
done